build.gradle

```diff
@@ -704,6 +704,9 @@ uiTestTargets.each { version ->
 
     if (project.hasProperty('headless')) {
       environment 'IDEPROBE_DISPLAY', 'xvfb'
+      // TODO (VirtusLab/ide-probe#195): remove the need to set these env variables
+      environment 'IDEPROBE_TEST_SUITE', 'uiTest' + version
+      environment 'IDEPROBE_TEST_CASE', 'exception'
```

Comment:
Note that this probably isn't a most fortunate name, as the screenshots might be stored for non-failed IntelliJ versions as well